Mathlib/FieldTheory/IntermediateField/Adjoin/Basic.lean

```diff
@@ -17,19 +18,7 @@ import Mathlib.LinearAlgebra.Dual
 /-!
 # Adjoining Elements to Fields
 
-In this file we introduce the notion of adjoining elements to fields.
-This isn't quite the same as adjoining elements to rings.
-For example, `Algebra.adjoin K {x}` might not include `x⁻¹`.
-
-## Main results
-
-- `adjoin_adjoin_left`: adjoining S and then T is the same as adjoining `S ∪ T`.
-- `bot_eq_top_of_rank_adjoin_eq_one`: if `F⟮x⟯` has dimension `1` over `F` for every `x`
-  in `E` then `F = E`
```

Comment:
These have disappeared.

Reply:
They are still around (the former is in defs and the latter is in basic).

Reply:
I can no longer find these docstrings; neither in the module docstring nor in decl docstring.

Reply:
(They are holdovers from the very early days of the file, but don't seem to be special enough for module docstrings anymore)

Reply:
Actually, the declarations already had docstrings, but I cleaned up the one for `adjoin_adjoin_left`.